src/coreclr/src/gc/gc.cpp

```diff
@@ -22268,12 +22197,12 @@ BOOL gc_heap::loh_size_fit_p (size_t size, uint8_t* alloc_pointer, uint8_t* allo
 {
     dprintf (1235, ("trying to fit %Id(%Id) between %Ix and %Ix (%Id)",
         size,
-        (2* AlignQword (loh_padding_obj_size) +  size),
+        (2* AlignQword (uoh_reloc_padding_obj_size) +  size),
```

Comment:
We have all kind of pading, often jus for alignment. The padding in loh (and possibly in poh) is for relocation purposes. It would be easier for me to understand why we have this padding if the name was a bit more descriptive of its purpose.
Will revert the way it was.

---
In reply to: [362298966](https://github.com/dotnet/runtime/pull/235#discussion_r362298966) [](ancestors = 362298966)